CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Circuitry 3.2.1 (April 5, 2018)
```

Comment:
I like the changelog inclusion, but I'd like to bump the version separately if you don't mind taking that aspect of it out of this PR.  (You could just change this to "Unreleased" or something in the interim.)

Reply:
@mhuggins cool. done.